src/RewardsStreamerMP.sol

```diff
@@ -444,7 +447,7 @@ contract RewardsStreamerMP is
             return 0;
         }
 
-        uint256 accruedMP = (timeDiff * account.stakedBalance * MP_RATE_PER_YEAR) / (365 days * SCALE_FACTOR);
```

Comment:
You need to remove scale factor if using math.muldiv

Reply:
Why?